hypha/apply/activity/templates/emails/activity_summary.md

```diff
@@ -0,0 +1,29 @@
+{% load humanize i18n %}
+
+{% trans "Dear" %},
+
+{% blocktranslate with total_count_apnumber=total_count|apnumber count counter=total_count %}
+You have {{ total_count_apnumber }} new activity:
+{% plural %}
+You have {{ total_count_apnumber }} new activities:
+{% endblocktranslate %}
+
+{% if submissions %}**{% trans "Submissions" %}**
+{% for msg in submissions %}
+- {{ msg.content_markdown }} • {{ msg.event.when }}{% endfor %}{% endif %}
+
+{% if comments %}**{% trans "Comments" %}**
+{% for msg in comments %}
+- {{ msg.content_markdown }} • {{ msg.event.when }}{% endfor %}{% endif %}
+
+{% if reviews %}**{% trans "Reviews" %}**
+{% for msg in reviews %}
+- {{ msg.content_markdown }} • {{ msg.event.when }}{% endfor %}{% endif %}
+
+{% if has_main_sections and messages %}**{% trans "Other activities" %}**{% endif %}
+{% for msg in messages %}
+- {{ msg.content_markdown }} • {{ msg.event.when }}{% endfor %}
+
+{% trans "Kind Regards" %},  
+{% trans "Team" %}  
+[{{org.NAME}}]{% if org.URL %}({{org.URL}}){% endif %}
```

Comment:
Would it make sense here to reuse the "base.html" template? Any maybe the template could reside with the other e-mail templates in "messages/email" dir?

Reply:
Done